tests/common/src/crypto_test.cpp

```diff
@@ -0,0 +1,677 @@
+/*
+ * Copyright (C) 2023 Renesas Electronics Corporation.
+ * Copyright (C) 2023 EPAM Systems, Inc.
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include <algorithm>
+#include <gtest/gtest.h>
+#include <memory>
+#include <vector>
+
+#include <mbedtls/ctr_drbg.h>
+#include <mbedtls/entropy.h>
+#include <mbedtls/error.h>
+#include <mbedtls/oid.h>
+#include <mbedtls/pk.h>
+#include <mbedtls/x509_crt.h>
+
+#include "aos/common/crypto/mbedtls/crypto.hpp"
+
+/***********************************************************************************************************************
+ * Static
+ **********************************************************************************************************************/
+static std::pair<aos::Error, std::vector<unsigned char>> GenerateRSAPrivateKey()
+{
+    mbedtls_pk_context       pk;
```

Comment:
For the tests we could use generated keys. But Im ok with this approach too.

Reply:
this security policy required
https://github.com/aoscloud/aos_core_lib_cpp/pull/59#discussion_r1466101084